curriculum/challenges/english/16-the-odin-project/top-introduction-to-flexbox/introduction-flexbox-lesson-d.md

```diff
@@ -1,12 +1,12 @@
 ---
 id: 6571b2ffcc1de61d7b4dd382
-title: Introduction to Flexbox Question D
+title: Introduction to Flexbox Lesson D
 challengeType: 15
-dashedName: introduction-flexbox-question-d
+dashedName: introduction-flexbox-lesson-d
 ---
 # --description--
 
-Now that you have a basic understanding of how flexbox works, what is the primairy role of a flex item in a `flexbox` layout? Try and answer the question below.
+Now that you have a basic understanding of how flexbox works, what is the primairy role of a flex item in a `flexbox` layout? Try and answer the lesson below.
```

Comment:
here wouldn't fit better `question`?

Reply:
naaaah

Reply:
So this should be reverted to `question`, correct?

It hasn't, so this doesn't seem resolved.

Reply:
You know what, I got my mind all mixed up with noodles.